build.rs

```diff
@@ -11,4 +12,7 @@ fn main() {
     if version().unwrap() >= Version::parse("1.82.0").unwrap() {
         println!("cargo:rustc-cfg=RUSTC_NEW_UNINIT_IS_STABLE");
     }
+    if version().unwrap() >= Version::parse("1.88.0-nightly").unwrap() {
```

Comment:
Note that not all 1.88.0-nightly versions support this, not sure if this is a problem or how to best handle this.
We could just bump this to 1.89-nightly once that exists.

Reply:
yeah that makes sense, created an issue: #40